src/mapping/tests/PolationTest.cpp

```diff
@@ -16,15 +16,17 @@ BOOST_AUTO_TEST_SUITE(Interpolation)
 BOOST_AUTO_TEST_CASE(VertexInterpolation)
 {
   PRECICE_TEST(1_rank);
-  mesh::Vertex vertex(Eigen::Vector3d(1.0, 2.0, 0.0), 0);
+  Eigen::Vector3d location(0.0, 0.0, 0.0);
```

Comment:
The unit test here is a bit trivial, maybe apply a different input in order to really cover the distance calculation.

Reply:
I added cases with a z difference. This will be the projection distance.